status/projects.py

```diff
@@ -138,19 +139,25 @@ def list_projects(self, filter_projects='all', oldest_date='2010-01-01', younges
 
 
         filtered_projects = OrderedDict()
+        # Specific list of projects given
+        if filter_projects[:1] == 'P':
+            fprojs = filter_projects.split(',')
+            for p_id, p_info in projects.iteritems():
+                if p_id in fprojs:
+                    filtered_projects[p_id] = p_info
+
```

Comment:
Yup, ok done - I think @remiolsen might have pointed out the same thing in a previous PR, but after it was merged?